hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardToolLocal.java

```diff
@@ -165,18 +165,19 @@ public void testImportNoFilesystem() throws Throwable {
 
   @Test
   public void testInfoBucketAndRegionNoFS() throws Throwable {
-    intercept(FileNotFoundException.class,
+    intercept(UnknownStoreException.class,
```

Comment:
if this is triggering failures, I'm not seeing them -but then I do have fs.s3a.bucket.probe == 0....it's probably not triggering any bucket probe at all. For consistent failures here, the config we run with should have that option set to 2, "use v2 probe on instantiation".

Reply:
v2 prove was used as a default value when I ran the test.
```
  public static final int S3A_BUCKET_PROBE_DEFAULT = 2;
```

The reaseon you did not see this may be that the ITestS3GuardToolLocal was just skipped. You are using DynamoDBMetadataStore rather than LocalMetadataStore?

Reply:
yeah, generally

Reply:
we should probably unset bucket probe options too & switch to the default (2) value; avoid differences in init behaviour

Reply:
This was addressed in `createConfiguration` above.